.travis.yml

```diff
@@ -114,7 +118,13 @@ install:
         BASEMAP="basemap"
         PYPROJ="pyproj"
       fi
-  - conda create -q -n test-environment
+  - |
+    if [[ "${PYTHON_VERSION}" == '3.7' ]]; then
+      FREETYPE="freetype"
+    else
+      FREETYPE="freetype<2.8"
+    fi
+  - conda create -q -n test-environment numpy scipy matplotlib basemap pyproj
```

Comment:
`numpy scipy matplotlib basemap pyproj` should already be set in the following lines, no need to add them again on this line